StripeCore/StripeCore/Source/API Bindings/STPAPIClient.swift

```diff
@@ -224,7 +227,7 @@ import UIKit
         guard let publishableKey = publishableKey, !publishableKey.isEmpty else {
             return false
         }
-        return publishableKey.lowercased().hasPrefix("pk_test")
+        return publishableKey.lowercased().hasPrefix("pk_test") || (publishableKeyIsUserKey && !userKeyLiveMode)
```

Comment:
This shows the "Test mode" badge when we're using testmode from the dashboard now

Reply:
Nice